docs/Deployment_guide.rst

```diff
@@ -0,0 +1,90 @@
+************************************
+Document to guide user to do changes for deployment of kubernetes feature in FIWARE Lab GUI
+************************************
+ 
+Fiware lab User can deploy Fiware GEs over kubernetes cluster by selecting the GEs from the catalogue provided in GUI. To access UI for deploying Fiware GE from horizon dashboard the user clicks on the kubernetes button in the horizon dashboard, they will get redirected to the UI Dashboard page where user will get options to create and manage kubernetes cluster and deploy Fiware GEs.
+To directky access the UI, the changes need to be made on the Horizon Dashboard code. This guide will show the necessary required changes. 
+
+Introduction:
+----------------------------------------------
+This Document provide step by step guide user to do changes in code of FIWARE Lab GUI and provide a button in Overview tab of FIWARE LAB GUI to redirect to kubernetes support GUI.
+
+**Step 1:** For deployment we have to do change in code so that we can get Project ID or Tenant ID and also Instance IP for all the instances on which user want to deploy kubernetes cluster.
+To do so we have to do changes in views.py Python file. 
+
+The path for the file is :- /horizon/openstack_dashboard/dashboards/project/overview/views.py.
+
+**Code Block:**
+
+from openstack_dashboard.dashboards.project.api_access.views \
+    import _get_openrc_credentials as get_pid
+
+from openstack_dashboard.dashboards.project.instances.views \
+    import IndexView
+
+from django.shortcuts import render, redirect
+import urllib.parse
+
+def kubernetes(request):
+    print("button clicked")
+    cred = get_pid(request)
+    pid = cred['tenant_id']
+    search_opts = {'marker': None, 'project_id': pid, 'limit': 21}
+    sort_dir = "desc"
+    indexView = IndexView()
+    indexView.request = request
+    instanceIP = []
+
+    instances = indexView._get_instances(search_opts=search_opts, sort_dir=sort_dir)
+    for i in range(len(instances)):
+        instanceIP.append(instances[i].addresses['shared'][0]['addr'])
+
+    a = ','.join(instanceIP)
+    url = "http://180.179.214.158:5000/auth/"+str(pid)+"?ip="+str(a)
+    return redirect(url)
+
+**Step 2:** For redirect we have to do changes in urls.py Python file so that user should be redirected to kubernetes support GUI and button should be functional. 
+
+The path for the file is :- /horizon/openstack_dashboard/dashboards/project/overview/urls.py.
+
+**Code Block:**
+
+urlpatterns = [
+    re_path(r'^$', views.ProjectOverview.as_view(), name='index'),
+    re_path(r'^warning$', views.WarningView.as_view(), name='warning'),
+    re_path(r'^kubernetes$', views.kubernetes, name='kubernetes_test'),
+]
+
+**Step 3:** To show kubernetes button in horizon dashboard of FIWARE Lab GUI we have to do changes in usage.html file.

```

Comment:
remove Python file, only file name should be there

Reply:
Changes were done